caasp-istio-proxyv2-image/Makefile

```diff
@@ -0,0 +1,10 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
This make file should be a symlink to `../.images.Makefile` as it is for the other images.

Reply:
@davidcassany Thanks for point this out. Will fix in the next update.